apiservermsgs/clustermsgs.go

```diff
@@ -182,6 +182,16 @@ type DeleteClusterResponse struct {
 	Status
 }
 
+// set the types for updating the Autofail status
+type UpdateClusterAutofailStatus int
+
+// set the different values around updating the autofail configuration
+const (
+	UpdateClusterAutofailDoNothing UpdateClusterAutofailStatus = iota
```

Comment:
You only need `iota` once.

Reply:
@abrightwell How do you initialize it then?

Reply:
```
const (
	UpdateClusterAutofailDoNothing UpdateClusterAutofailStatus = iota
	UpdateClusterAutofailEnable
	UpdateClusterAutofailDisable
)

Reply:
I was testing it and thought it was that way, but got an error. The error turned out to be at a different line fo code